anesthetic/plot.py

```diff
@@ -526,6 +536,13 @@ def kde_contour_plot_2d(ax, data_x, data_y, *args, **kwargs):
         A set of contourlines or filled regions
 
     """
+    kwargs = cbook.normalize_kwargs(kwargs,
+                                    dict(linewidths=['linewidth', 'lw'],
+                                         linestyles=['linestyle', 'ls'],
+                                         color=['c'],
+                                         facecolor=['fc'],
+                                         edgecolor=['ec']),
+                                    forbidden=['colors'])
```

Comment:
I'm getting quite a lot of the warning message:
```python
MatplotlibDeprecationWarning:             
The 'forbidden' parameter of normalize_kwargs() was deprecated in Matplotlib 3.3 and will be removed two minor releases later. If any parameter follows 'forbidden', they should be passed as keyword, not positionally.                                                                           
```

Is this something that we should be worried about?

Reply:
Maybe, I've been swamped with matplotlib warnings lately (probably due to my stylesheet going out of date) that I didn't notice this particular one. Will look into it.

Reply:
I removed the `forbidden` parameter. This wasn't crucial anyhow, I just put it in as a reminder/early catch that anesthetic does not work with kwarg `colors`. This is because contour plots only allow either `cmap` or `colors` as input, not both. This is not a new thing of this PR.